### DISCUSSION TITLE
Managing large Libraries

### DISCUSSION THREAD

**Discussion by Cniewiad** [community/community#31978] [Other Feature Feedback, Questions, & Ideas] [answered]
Guys, I am a new user of the Dopamin and I really like it.
I have been using it now for over 2 weeks, however, I am having problems.

One of the reasons that I started using it, is that I have got a large library of music. I have got several thousand CD's and LP;s that I have converted to digital format and now I have got over 460,000 songs. I used a lot of different managers Monkey, MusicBee, Helium, AIMP, foobar2000 just to name a few. They all have good things but also some of them are not up to scratch for my need to organize music.
When I saw Dopelmin I liked it straight away.
However, there is one glitch that I think can be easily fixed. With any operation on the files refreshing the library takes 55 sec. before I can start using a program again. Not sure why, as Monkey, musicBee or foobar they are very fast with the same size library - merely 3-4 sec.
As I said I like Dopeline very much, but with such lagging for me is not usable.
I wonder if this is something that can be easily fixed.
Chris

**Comment by airtower-luna** [comment, marked as answer]
It looks like you're looking for help with a software hosted on GitHub, maybe https://github.com/digimezzo/dopamine? In that case you should open an issue *in that repository* so the developers see it. :slightly_smiling_face:

**Reply by queenofcorgis** [reply]
Jumping in to add,  the best way to contact the maintainers of that software is to:

- Check the README for instructions on how to operate the software and pointers to documentation or troubleshooting info.
- Check the [SUPPORT file,](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project) if one exists, for instructions on how to best contact the maintainers for support.
- Check the [CONTRIBUTING guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.

All of these documents can be found, if they exist, in the repository where you found the software itself.

**Comment by doruk5** [comment]
![Uploading Screenshot_20231129-093909_Telegram.jpg…]()

**Reply by doruk5** [reply]
Dolandirildim.yardim.istiyorum
